helm/logging-operator/templates/deployment.yaml

```diff
@@ -37,7 +37,10 @@ spec:
         {{- else -}}
           - -enable-logging=false
         {{- end }}
-        - -installation-name={{ .Values.loggingOperator.installationName }}
+        - -installation-name={{ .Values.managementCluster.name }}
```

Comment:
This requires a change in `config` repo.

Reply:
Yep it's done

Reply:
Let me track it After lunch

Reply:
Sée https://github.com/giantswarm/config/blob/b95dba75f82525f9ece4141b4eb312d4f3c4902e/default/apps/logging-operator/configmap-values.yaml.template#L2